modules/client/src/main/java/org/jboss/wsf/stack/cxf/i18n/Loggers.java

```diff
@@ -257,4 +257,12 @@ public interface Loggers extends BasicLogger
     @LogMessage(level = ERROR)
     @Message(id = 24115, value = "Failed to compute UsernameToken profile digest from expected password")
     void failedToComputeUsernameTokenProfileDigest();
+
+    @LogMessage(level = TRACE)
+    @Message(id = 24116, value = "Plain text password must be recoverable when UsernameToken Profile is used with PasswordDigest")
+    void plainTextPasswordMustBeRecoverable();
```

Comment:
Just some minor thing , can we add some cause parameter to this log message ? Does this help give more error details ? 

@Message(id = 24116, value = "Plain text password for principal: %s must be recoverable when UsernameToken Profile is used with PasswordDigest")
void plainTextPasswordMustBeRecoverable(String principal, @Cause Throwable cause);

Reply:
Updated